Yosemite/YosemiteTests/Stores/AppSettingsStoreTests.swift

```diff
@@ -919,10 +919,8 @@ extension AppSettingsStoreTests {
         try fileStorage?.deleteFile(at: expectedGeneralStoreSettingsFileURL)
 
         let currentTime = Date()
-        let datePrior = Date(timeIntervalSince1970: 100)
-        let date = Date()
```

Comment:
This has the same value as `currentTime` above, `Date()`. It should be enough to reuse `currentTime` where `date` was used.